admin-dev/themes/default/template/nav.tpl

```diff
@@ -4,72 +4,74 @@
 		<i class="material-icons">chevron_left</i>
 	</span>
 
-	<ul class="main-menu">
-		{foreach $tabs as $level_1}
-			{if $level_1.active}
-				{* Dashboard exception *}
-				{if $level_1.class_name == 'AdminDashboard'}
-					<li class="link-levelone {if $level_1.current}-active{/if}" id="tab-{$level_1.class_name}" data-submenu="{$level_1.id_tab}">
-						<a href="{if $level_1.sub_tabs|@count && isset($level_1.sub_tabs[0].href)}{$level_1.sub_tabs[0].href|escape:'html':'UTF-8'}{else}{$level_1.href|escape:'html':'UTF-8'}{/if}" class="link" >
-							<i class="material-icons">{$level_1.icon}</i>
-							<span>{if $level_1.name eq ''}{$level_1.class_name|escape:'html':'UTF-8'}{else}{$level_1.name|escape:'html':'UTF-8'}{/if}</span>
-						</a>
-					</li>
-				{else}
-					<li class="category-title {if $level_1.current}-active{/if}" id="tab-{$level_1.class_name}" data-submenu="{$level_1.id_tab}">
-						<span class="title">
-							<span>{if $level_1.name eq ''}{$level_1.class_name|escape:'html':'UTF-8'}{else}{$level_1.name|escape:'html':'UTF-8'}{/if}</span>
-						</span>
-					</li>
+  <div class="nav-bar-overflow">
```

Comment:
@NeOMakinG @PierreRambaud  Sorry for being late on this but I just noticed it while merging `1.7.7.x` into `develop`
Isn't it a BC break ? adding a `<div class="nav-bar-overflow">` on top of the menu ?
This will break any module JS script using selectors ?

Reply:
Depending how it is selected, but even if it's a BC Break, thats a bug which still block the use of the menu while on responsive device, that's pretty huge, it means that you can't use your backoffice on responsive device, that should be fixed even introducing BC

Reply:
If it's a BC Break, it must defined in changelog ;)